.vscode/launch.json

```diff
@@ -65,6 +65,39 @@
       "environment": [],
       "externalConsole": true,
       "preLaunchTask": "build"
+    },
+    {
+      "name": "(Windows) Launch library_test (one)",
+      "type": "cppvsdbg",
+      "request": "launch",
+      "program": "${workspaceFolder}/build/x64-Debug/bin/library_test.exe",
+      "args": [
+        "--gtest_filter=${input:test_pattern}"
+      ],
+      "stopAtEntry": false,
+      "cwd": "${workspaceFolder}/build/x64-Debug/bin/",
+      "environment": [],
+      //"externalConsole": true,
+      "preLaunchTask": "build"
+    },
+    {
+      "name": "(Windows) Attach",
+      "type": "cppvsdbg",
+      "request": "attach",
+      "processId": "${command:pickProcess}",
+      "args": [],
```

Comment:
This might be related to me not having Windows environment anymore but there is this warning:

![image](https://github.com/user-attachments/assets/b094835e-0ba6-41a3-b79d-8cfffd722b35)

Reply:
Given this is an `attach` type entry, it would make sense that `args` is not allowed, I'll look into it.